includes/Core/Admin/Screens.php

```diff
@@ -482,6 +466,22 @@ private function get_screens() {
 			)
 		);
 
+		// Wrap this simply to save some unnecessary filter firing and screen instantiation.
+		if ( current_user_can( Permissions::VIEW_MODULE_DETAILS ) ) {
+			/**
+			 * Filters the admin screens for modules.
+			 *
+			 * By default this is an empty array, but can be expanded.
+			 *
+			 * @since 1.0.0
+			 *
+			 * @param array $module_screens List of Screen instances.
+			 */
+			$module_screens = apply_filters( 'googlesitekit_module_screens', array() );
+
+			$screens = array_merge( $screens, $module_screens );
+		}
```

Comment:
@aaemnnosttv What is the change in this file about? Why did you change the order of the settings screen and module screens (which are still relevant in theory for pre-`unifiedDashboard`)?